distribution/docs/src/main/jdocs/content/_security-framework/security-pep-intro-contents.adoc

```diff
@@ -0,0 +1,10 @@
+:title: Security PEP
+:type: securityFramework
+:status: published
+:parent: Security Framework
+:children: Security PEP Interceptor
+:order: 06
+:summary: Security PEP.
+
+The ${ddf-security} Policy Enforcement Point (PEP) application contains bundles and services that enable service and metacard authorization.
```

Comment:
❓ What is "metacard authorization"?

Reply:
I'll get some input from @stustison on that one

Reply:
Not really true that they can be installed separately anymore. We might just say that it contains bundles that allow for policies to be enforced at various parts of the system, for example: to reach contexts, view metacards, access catalog operations, and others.